scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -165,26 +95,46 @@
    "outputs": [],
    "source": [
     "# load TIFFs of segmentation labels (from pixel clustering, deepcell, etc.)\n",
-    "segmentation_labels_preload = data_utils.tiffs_to_xr_labels(tiff_dir=label_dir, output_dir=None, delimiter='_')\n",
-    "plt.figure(figsize = (13, 13))\n",
-    "plt.imshow(segmentation_labels_preload.loc[\"Point8\", :, :, \"segmentation_label\"])"
+    "segmentation_labels = data_utils.tiffs_to_xr_labels(tiff_dir=label_dir, output_dir=None, delimiter='_feature_0')\n",
+    "if plot_labels:    \n",
+    "    for fov in segmentation_labels.fovs:\n",
+    "        plt.figure(figsize = (13, 13))\n",
```

Comment:
Does this work? I would assume a for loop over imshow would just overwrite the previous image, but I didn't test it out

Reply:
yeah it might just overwrite the prev image.  I'll try deliberately creating new figs to send data to (might also have to change `plot_utils.plot_overlay` to take in an axes)

Reply:
I think we can just remove this line completely. The overlays are going to be more informative.